src/main/java/com/bookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -39,6 +40,14 @@ public ResponseEntity<Object> handleRegistrationException(RegistrationException
         return new ResponseEntity<>(error, HttpStatus.CONFLICT);
     }
 
+    @ExceptionHandler(JwtException.class)
```

Comment:
JwtException won't be handled by GlobalHandler becuase it appears in filter layer